webtau-http-groovy/src/main/groovy/org/testingisdocumenting/webtau/http/HttpGroovyConfig.groovy

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 webtau maintainers
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.testingisdocumenting.webtau.http
+
+import org.testingisdocumenting.webtau.cfg.WebTauConfig
+import org.testingisdocumenting.webtau.cfg.WebTauConfigHandler
+import org.testingisdocumenting.webtau.http.listener.HttpListener
+import org.testingisdocumenting.webtau.http.listener.HttpListeners
+
+import static org.testingisdocumenting.webtau.utils.ConfigUtils.instancesFromClassList
+
+class HttpGroovyConfig implements WebTauConfigHandler {
+    @Override
+    void onAfterCreate(WebTauConfig cfg) {
+        List<HttpListener> listeners = instancesFromClassList(cfg.get('httpListeners'))
+        listeners.each { HttpListeners.add(it) }
```

Comment:
to be consistent, let's either move all the other groovy file configs out or put this one in. For now my preference would be to put all in as they are one liners mostly

Reply:
You mean move this config to `WebTauGroovyFileConfigHandler`? I was going to do that initially but in my mind had the webtau-groovy and webtau-http-groovy dependency the wrong way round so didn't think it would be possible.  I'll move it, I agree it makes sense for it to be in one place.

Reply:
did you push?

Reply:
I have now.